### ISSUE TITLE
Although I provided 10 sequences, only two of them were clustered. What happened to the other 8 sequences?

### ISSUE THREAD

**Issue by Y-antian** [joybio/multiPrime#13]
my input:
![1710139079140](https://github.com/joybio/multiPrime/assets/130638302/528c9719-3e69-4ed7-b27f-2863b61196dc)
my cluster:
![1710139124092](https://github.com/joybio/multiPrime/assets/130638302/3f3cf101-ae3a-44fb-b998-cf403409f3e4)

**Comment by Y-antian** [comment]
It appears that the multiPrime function only utilizes the portion of the ID before the "-" symbol.

**Comment by joybio** [comment]
Yes, that's correct. The new updated version of multiPrime does not permit the use of colons (:), hyphens (-), or spaces in the FASTA IDs. If any of these characters exist, only the first element before these separators will be considered as the ID.
